### PR TITLE
🔥 Semantic ui style cleanup

### LINKED ISSUE #381: Change study table header text to be sentence case
Change from : 
Screen Shot 2019-07-22 at 11 16 43 AM
Screen Shot 2019-07-22 at 11 16 55 AM

Change to:
Screen Shot 2019-07-22 at 11 17 49 AM
Screen Shot 2019-07-22 at 11 18 02 AM

### PR DESCRIPTION
**Pull request by XuTheBunny**
- No inline-style left. 
- No unused CSS class left. 
- No console warning. 

Closes #381

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -124,3 +55,19 @@ body {
   margin-top: 0px;
   margin-right: 20px;
 }
+
+.FileList--row {
```
Comment (bdolly) [inline src/App.css]: let's go with `.cursor-pointer` instead to make it more general and reusable

File: src/components/StudyList/StudyTable.js [hunk 2]
```diff
@@ -99,7 +99,14 @@ const StudyTable = ({
       <Table.Header>
         <Table.Row>
           {cols.map(v => (
-            <Table.HeaderCell key={v}>{v}</Table.HeaderCell>
+            <Table.HeaderCell key={v}>
+              {v === 'kfId'
+                ? 'Study ID'
+                : v
+                    .replace(/([A-Z])/g, ' $1')
```
Comment (dankolbman) [inline src/components/StudyList/StudyTable.js]: This may be able to move to it's own function at the least.

File: src/App.css [hunk 3]
```diff
@@ -124,3 +55,19 @@ body {
...
+/* Needed so that size does not change when showing success icon */
+.CopyButton--popup {
+  width: 4rem;
+  text-align: center;
+}
+
+.View--Login {
+  height: 100vh;
+  background-image: linear-gradient(to bottom, #fff 400px, transparent 100%),
+    url(https://portal.kidsfirstdrc.org/static/media/background-science.68317e4e.jpg);
```
Comment (bdolly) [inline src/App.css]: needs to be a local image in the project and not pulled from a poral url

File: src/components/FileDetail/FileDetail.js [hunk 4]
```diff
@@ -173,7 +173,7 @@ const FileDetail = ({ fileNode, history, match }) => {
                     </Segment.Group>
 
                   </Grid.Column>
-                  <Grid.Column mobile={8} tablet={8} computer={3}>
+                  <Grid.Column mobile={8} tablet={4} computer={3}>
```
Comment (dankolbman) [inline src/components/FileDetail/FileDetail.js]: There's probably some more work to do with making the actions more responsive in the future. Maybe inline them on tablet devices.

**Pull request merged**